### ISSUE TITLE
App install experience not working correctly in High contrast mode 

### ISSUE THREAD

**Issue by ianyyu** [microsoft/devhome#1375]
### Dev Home version

0.301.198.0

### Windows build number

10.0.22621.1992

### Other software

OS Build Version: 10.0.22621.1992.amd64fre.ni_release.220506-1250
.NET Version: .NET 6.0.18

### Steps to reproduce the bug

turn on high contrast mode and navigate to the App install experiences 

### Expected result

Follow the selected high contrast theme 

### Actual result

Red brushes applied to backplates and strokes 

![image](https://github.com/microsoft/devhome/assets/50185601/0f978e85-4864-461a-94b2-3d138996d539)

### Included System Information

Physical Memory: 15.95GB (10.19GB free)
Processor Architecture: x64

### Included Extensions Information

Extensions: 
Microsoft.Windows.DevHomeGitHubExtension_0.300.198.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHome_0.301.198.0_x64__8wekyb3d8bbwe

**Comment by ianyyu** [comment]
@AmelBawa-msft

**Comment by AmelBawa-msft** [comment]
Issue possibly related to missing the `Brush` suffix in the `ThemeResource` for the controls background.
